ui/js/directives/treeherder/resultsets.js

```diff
@@ -47,7 +47,7 @@ treeherder.directive('thAuthor', function () {
             scope.authorName = userTokens[0].trim();
             scope.authorEmail = email;
         },
-        template: '<span title="View resultsets for {{authorName}}: {{authorEmail}}">' +
+        template: '<span title="View pushes for {{authorName}}: {{authorEmail}}">' +
```

Comment:
What do you think about "View pushes by {{authorEmail}}"? I think the author name is redundant (after all it's the email that goes in the URL filter param).